pkg/security/podsecuritypolicy/sysctls/mustmatchpatterns.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sysctls
+
+import (
+	"fmt"
+
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/util/validation/field"
+	"strings"
+)
+
+// mustMatchPatterns implements the CapabilitiesStrategy interface
+type mustMatchPatterns struct {
+	patterns []string
+}
+
+var (
+	_ SysctlsStrategy = &mustMatchPatterns{}
+
+	defaultSysctlsPatterns = []string{"*"}
+)
+
+// NewMustMatchPattrens creates a new mustMatchPattern strategy that will provide validation.
+func NewMustMatchPatterns(patterns []string) (SysctlsStrategy, error) {
+	if patterns == nil {
+		patterns = defaultSysctlsPatterns
```

Comment:
heh, just realized this isn't against master 😄  guess we can wait for feedback when you submit there.